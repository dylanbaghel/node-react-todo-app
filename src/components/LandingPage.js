import React from 'react';
import { Link } from 'react-router-dom';

const LandingPage = () => {
    return (
        <div className="container text-center d-flex flex-column justify-content-center landing">
            <h1>Welcome To Todo App</h1>
            <p>Created By Abhishek Baghel</p>
            <div className="d-flex justify-content-center">
                <Link to="/login" className="btn btn-primary mr-1">Login</Link>
                <Link to="/signup" className="btn btn-danger ml-1">Sign Up</Link>
            </div>
        </div>
    );
};

export default LandingPage;