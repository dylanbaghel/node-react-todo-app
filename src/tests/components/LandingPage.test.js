import React from 'react';
import { shallow } from 'enzyme';

import LandingPage from './../../components/LandingPage';

test('should render landing page correctly', () => {
    const wrapper = shallow(<LandingPage />);
    expect(wrapper).toMatchSnapshot();
});