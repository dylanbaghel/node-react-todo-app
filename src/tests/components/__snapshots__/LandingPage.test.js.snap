// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render landing page correctly 1`] = `
<div
  className="container text-center d-flex flex-column justify-content-center landing"
>
  <h1>
    Welcome To Todo App
  </h1>
  <p>
    Created By Abhishek Baghel
  </p>
  <div
    className="d-flex justify-content-center"
  >
    <Link
      className="btn btn-primary mr-1"
      replace={false}
      to="/login"
    >
      Login
    </Link>
    <Link
      className="btn btn-danger ml-1"
      replace={false}
      to="/signup"
    >
      Sign Up
    </Link>
  </div>
</div>
`;
